pywebpack/__init__.py

```diff
@@ -7,7 +7,289 @@
 # it under the terms of the Revised BSD License; see LICENSE file for
 # more details.
 
-"""Webpack integration layer for Python."""
+r"""Webpack integration layer for Python.
+
+A simple example
+----------------
+
+Let's take a simple ``package.json`` that includes a Node module::
+
+    {
+      "private": true,
+      "name": "example",
+      "version": "0.0.1",
+      "author": "myself",
+      "license": "WTFPL",
+      "description": "example",
+      "dependencies": {
+        "jquery": "^3.2.1"
+      }
+    }
+
+And let's create a Webpack configuration file ``webpack.config.js`` to load the
+entry point ``src/index.js`` and output the built version in the ``dist``
+folder::
+
+    var path = require('path');
+
+    module.exports = {
+      context: path.resolve(__dirname, 'src'),
+      entry: './index.js',
+      output: {
+        filename: '[name].js',
+        path: path.resolve(__dirname, 'dist')
+      }
+    };
+
+Now, let's add to the ``packages.json`` a run script ``build`` that executes
+webpack::
+
+    "scripts": {
+      "build": "webpack --config webpack.config.js"
+    }
+
+We can easily wrap our project in a :class:`~pywebpack.project.WebpackProject`
+object::
+
+    from pywebpack.project import WebpackProject
```

Comment:
``from pywebpack import WebpackProject``

The entire API is exposed in ``pywebpack`` to have simple import statements.